### PR TITLE
build(multiview-triangle-view): use node16 module resolution

### PR DESCRIPTION
**Pull request by DLehenbauer**
Semi-automatic PR to switch to node16 module resolution

### CODE REVIEW

File: examples/data-objects/multiview/triangle-view/package.json [hunk 1]
```diff
@@ -11,14 +11,22 @@
...
 	"license": "MIT",
 	"author": "Microsoft and contributors",
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			}
+		}
```
Comment (tylerbutler) [inline examples/data-objects/multiview/triangle-view/package.json:21]:
```suggestion
	"exports": {
		".": {
			"types": "./lib/index.d.ts",
			"default": "./lib/index.js"
		}
```

File: examples/data-objects/multiview/triangle-view/package.json [hunk 2]
```diff
@@ -34,9 +42,6 @@
 		"start:spo": "webpack serve --config webpack.config.cjs --env mode=spo",
 		"start:spo-df": "webpack serve --config webpack.config.cjs --env mode=spo-df",
 		"start:tinylicious": "webpack serve --config webpack.config.cjs --env mode=tinylicious",
-		"test": "npm run test:jest",
-		"test:jest": "jest",
-		"test:jest:verbose": "cross-env FLUID_TEST_VERBOSE=1 jest",
```
Comment (tylerbutler) [inline examples/data-objects/multiview/triangle-view/package.json:39]: Intentional?

File: examples/data-objects/multiview/triangle-view/tests/view.test.ts
Comment (tylerbutler) [inline examples/data-objects/multiview/triangle-view/tests/view.test.ts:1]: Oh I see, tests were lies... :)

File: examples/data-objects/multiview/triangle-view/package.json (hunk 1 above)
Comment (DLehenbauer) [reply examples/data-objects/multiview/triangle-view/package.json:21]: @tylerbutler - Do you mind if we simplify the exports in a separate pass?

(Consistency is easier for my lamely written conversion and reporting scripts.)

File: examples/data-objects/multiview/triangle-view/tests/view.test.ts
Comment (DLehenbauer) [reply examples/data-objects/multiview/triangle-view/tests/view.test.ts:1]: Yeah.  :-)

File: examples/data-objects/multiview/triangle-view/package.json (hunk 2 above)
Comment (DLehenbauer) [reply examples/data-objects/multiview/triangle-view/package.json:39]: Yeah.  (I see you figured it below.  :-)

File: examples/data-objects/multiview/triangle-view/package.json (hunk 1 above)
Comment (tylerbutler) [reply examples/data-objects/multiview/triangle-view/package.json:21]: Mentioned elsewhere but for completeness: yes, waiting to clean these up in a pass later is fine.

**Review by tylerbutler — APPROVED** [review]
Looks good. Probably worth merging latest main and resetting/regenning the lockfile before merge.

**Review by jason-ha — CHANGES_REQUESTED** [review]

File: examples/data-objects/multiview/triangle-view/webpack.config.cjs [hunk 3]
```diff
@@ -38,7 +38,7 @@ module.exports = (env) => {
 			},
 			output: {
 				filename: "[name].bundle.js",
-				path: path.resolve(__dirname, "dist"),
+				path: path.resolve(__dirname, "lib"),
```
[the remark this answers is not in the document]
Comment (DLehenbauer) [reply examples/data-objects/multiview/triangle-view/webpack.config.cjs]: Thanks.  I've stopped changing webpack output, but this one slipped through.

**Review by jason-ha — APPROVED** [review]

**Comment by jason-ha** [comment]
@DLehenbauer, what is left that wasn't covered by #20085?

**Comment by DLehenbauer** [comment]
Merged with PR #20085

**Comment by jason-ha** [comment]
@tylerbutler, GitHub lets us commit no change PRs? https://github.com/microsoft/FluidFramework/commit/1d079029ce705a43b2905f62a4bcc0b0ffad6aee

**Comment by tylerbutler** [comment]
> @tylerbutler, GitHub lets us commit no change PRs? https://github.com/microsoft/FluidFramework/commit/1d079029ce705a43b2905f62a4bcc0b0ffad6aee

Looks like it. I'm guessing we could prevent this by requiring an up to date base branch but that's been a tough sell so far.

**Pull request merged**